specs/cloud-defend.spec.yml

```diff
@@ -3,10 +3,16 @@ inputs:
   - name: cloud_defend/control
     description: "Defend for containers"
     platforms: &platforms
+      - linux/amd64
+      - linux/arm64
       - container/amd64
       - container/arm64
     outputs: &outputs
       - elasticsearch
+    runtime:
+      preventions:

```

Comment:
@rhysre Looks like you can check other criteria other than root access. see [specs/endpoint-security.spec.yml](https://github.com/elastic/elastic-agent/blob/main/specs/endpoint-security.spec.yml#L17)

not sure if this is needed, but thought i'd check with you.